### PR TITLE
Adds an all versions page for packages

### PR DESCRIPTION
**Pull request by supersimple**
Fixes https://github.com/hexpm/hexpm/issues/727

screen shot 2018-12-03 at 11 00 30 am

URLs are:
/packages/:repo/:pkgname/versions
/packages/:pkgname/versions

### CODE REVIEW

File: lib/hexpm_web/templates/versions/index.html.eex [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<h2 class="versions-title">
+  All available versions of <span class="version-package-title"><%= package_name(@package) %></span>
```
Comment (wojtekmach) [inline lib/hexpm_web/templates/versions/index.html.eex]: How about linking package name to package page?

```suggestion
  All available versions of <%= link package_name(@package), to: path_for_package(@package), class: "version-package-title" %>
```

File: lib/hexpm_web/templates/package/show.html.eex [hunk 2]
```diff
@@ -134,9 +134,8 @@ tools = Keyword.take(tools, compatible_tools)
             <%= render "versions.html", [package: @package, releases: Enum.drop(@releases, show_latest_versions)] %>
           </ul>
```
Comment (ericmj) [inline lib/hexpm_web/templates/package/show.html.eex]: Should we still render the collapsed version list?

**Comment by ericmj** [comment]
Looks good, please look at the ebert comments but only if they make sense to follow, they are not critical. 👍

**Comment by supersimple** [comment]
@ericmj those ebert warnings are outdated now. I should have fixed both.

**Pull request merged**